builder/frameworks/_embed_files.py

```
@@ -21,6 +21,7 @@
 Import("env")
 
 board = env.BoardConfig()
+is_xtensa = mcu in ("esp32","esp32s2","esp32s3")
```

Ruff (0.12.2)

Undefined name `mcu`

(F821)